newsletters/033.md

```diff
@@ -18,7 +18,7 @@ Most of the code is already open-source and can be obtained from repositories su
 
 + Linux Official Community Upstream work:
 
-  + 
+  + 6.8 is officially released, and 6.9 enters the merge window. The dts part of the reset driver has been modified into the mainline, and other driver/bindings parts should be merged soon.
```

Comment:
has been merged into the mainline / has landed in the mainline?

Reply:
corrected and pushed again.